+nosnoc/+time_freezing/cls_elastic.m

```diff
@@ -0,0 +1,74 @@
+function [pss_model] = cls_elastic(cls_model, opts)
+    dims = cls_model.dims;
+    pss_model = nosnoc.model.Pss();
+    pss_model.dims = dims;
+
+    % transfer all common properties to pss_model
+    common_props = properties('nosnoc.model.Base');
+    for ii=1:length(common_props)
+        prop = common_props{ii};
+        pss_model.(prop) = cls_model.(prop);
+    end
+    
+    dims.n_contacts = length(cls_model.f_c);
```

Comment:
only n_contact=1 is supported for the automated elastic, right?

Reply:
Yep. added an error.